client/src/reader/containers/annotation/annotatable-components/Drawer.js

```diff
@@ -33,22 +24,12 @@ export default class AnnotatableDrawer extends PureComponent {
     return upperFirst(this.props.drawerState);
   }
 
-  setDrawerWrapperProps = drawerWrapperProps => {
```

Comment:
Please explain why this component is being changed in this way.

Reply:
According to eslint:
`Unused state field: 'drawerWrapperProps'  react/no-unused-state`

Looking at the file, I could not find any reference to the `drawerWrapperProps` and it seemed safe to delete it.

Upon closer inspection, the drawer wrapper props were intended to be passed into the `Drawer.Wrapper` as `<Drawer.Wrapper {...drawerWrapperProps}>` instead of the current `<Drawer.Wrapper {...drawerProps}>`.